src/avram/query.cr

```diff
@@ -1,7 +1,8 @@
 require "./queryable"
 
 abstract class Avram::Query
-  def destroy_all
+  # runs a SQL `TRUNCATE` on the current table
+  def truncate
```

Comment:
Could you make this a class method as well? Since it ignores any other conditions that might be good